### ISSUE TITLE
BUG WHEN UPDATE MAC OS BIG SUR

### ISSUE THREAD

**Issue by aliceab1** [horosproject/horos#657]
Hi,
When i updated my MacBook Pro to BIG SUR, horus 3.3.6 that was perfect fine before became very slow in scrolling images, specially from CT scans. To bad. So I updated for the last version thats suposed to work on Big Sur, and the same BUG is there.

**Comment by captaincrunch93** [comment]
this seems like a known bug - I've been having it as well. a temp work around is to use the orthogonal MPR recon. accessed when you're in the 2d viewer through the 3d viewer dropdown menu. Seems to work much more smoothly, bit of a hassle but a work around while we wait for a fix.

**Comment by rawtherapy** [comment]
yes, it's about a year that this issues keep going on big sur.. hope in a solution from Horos team! ;)
